Calibration/IsolatedParticles/interface/FindCaloHit.h

```diff
@@ -34,6 +34,8 @@ namespace spr {
   template <typename T>
   std::vector<typename T::const_iterator> findHit(edm::Handle<T>& hits, DetId thisDet, bool debug=false);
 
+  std::vector<std::vector<PCaloHit>::const_iterator> findHit(std::vector<PCaloHit>& hits, DetId thisDet, bool debug=false);
```

Comment:
Is this specialization really needed? From a quick look, it seems like you just write down what is done also in the general case.